apps/channels/views.py

```diff
@@ -124,3 +127,32 @@ def new_api_message(request, experiment_id: uuid):
         session,
     )
     return Response(data={"response": response})
+
+
+@require_POST
+@csrf_exempt
+@verify_hmac
+def new_connect_message(request: HttpRequest):
+    serializer = CommCareConnectMessageSerializer(data=json.loads(request.body))
+    if not serializer.is_valid():
+        return JsonResponse(serializer.errors, status=400)
+
+    connect_channel_id = serializer.data["channel_id"]
+    try:
+        participant_data = ParticipantData.objects.prefetch_related("participant").get(
```

Comment:
unnecessary prefetch
```suggestion
        participant_data = ParticipantData.objects.get(
```